modules/darwinosl/CMakeLists.txt

```diff
@@ -26,11 +26,9 @@ if(APPLE)
     #include <OSLog/OSLog.h>
     int main(int, char**)
     { return 0; }
-  " HAVE_OSLLOG)
+  " HAVE_OSLOG)
```

Comment:
and, then also make this whole test guarded by the status of ENABLE_DARWIN_OSL

Reply:
Is there any point in enabling ObjC but disabling OSLog? The whole thing sits inside `if(ENABLE_OBJC)` , that is why I saw no point in a second macro.
Or we want to make these independent?